### PR TITLE
fix(notice): fixes colors in notice when used without parent theme component

### PR DESCRIPTION
**Pull request by pretzelhammer**
## Describe the problem this PR addresses

success, warning, & info notices used without a parent mtheme component had red background colors

## Describe the changes in this PR

fixes the colors

## Other information

nope

### CODE REVIEW

File: src/components/Notice/src/Notice.vue
Comment (landondurnan) [inline src/components/Notice/src/Notice.vue:152]: typo

**Review by landondurnan — COMMENTED** [review]
Do you know what introduced this issue?

**Comment by pretzelhammer** [comment]
@landondurnan "v9" is 32 separate beta releases that happened over 10 months rolled into a single release. If we need to track something down in "v9" we need to look at all of the beta releases from v7 to v9.

It was still working fine in [v7-beta.6](https://square.github.io/maker/styleguide/7.0.0-beta.6/#/Notice). Something weird and unexplainable happened in [v7-beta.7](https://square.github.io/maker/styleguide/7.0.0-beta.7/#/Notice) (probably the styleguide build broke). It broke in [v7-beta.8](https://square.github.io/maker/styleguide/7.0.0-beta.8/#/Notice) with [this commit](https://github.com/square/maker/commit/a8da0b341e8fdadd0287dd55155f21545b38d7a8#diff-072cc9e1481b797412eb201a1a5520c6ffbfb6e2d79de3924bde71255cd2ef50) and was still broken but in a slightly different way in [v7-beta.11](https://square.github.io/maker/styleguide/7.0.0-beta.11/#/Notice) with [this commit](https://github.com/square/maker/commit/f560b3bb194f0b899870e69a26e16305708d99b2#diff-072cc9e1481b797412eb201a1a5520c6ffbfb6e2d79de3924bde71255cd2ef50).

It's not clear to me why the notice icons went from being filled to outlined, and why the individual type background colors were removed, they're still both in the Maker Figma. I think these changes were made as part of Lauren's color profiles work in Maker and Website and she'll probably know the motivations behind them and is the best person to review this PR to make sure it doesn't break anything in Website. I'll wait for a review from her.

**Review by laurenhu — APPROVED** [review]

**Pull request merged**
